datasets/samplemotordatalimerick.py

```diff
@@ -0,0 +1,671 @@
+
+    # ds_ratio = 10,  sr: 20K / 10 = 2000, 0.25 sec window, fft input will have: 500 samples,
+    # fftout's first 256 samples will be used
+    # cnn input will have 256 samples
+
+    accel_in_second_dim = True
+
+    random_or_speed_split = True
+
+    return samplemotordatalimerick_get_datasets(data, load_train, load_test,
+                                                downsampling_ratio=downsampling_ratio,
+                                                signal_duration_in_sec=signal_duration_in_sec,
+                                                overlap_ratio=overlap_ratio,
+                                                eval_mode=eval_mode,
+                                                label_as_signal=label_as_signal,
+                                                random_or_speed_split=random_or_speed_split,
+                                                accel_in_second_dim=accel_in_second_dim)
+
+
+def samplemotordatalimerick_get_datasets_for_eval_with_anomaly_label(data,
+                                                                     load_train=True,
+                                                                     load_test=True):
+    """"
+    Returns Sample Motor Data Limerick Dataset For Evaluation Mode
+    Label is anomaly status
+    """
+
+    eval_mode = True   # Test set includes validation normals
+    label_as_signal = False
+
+    signal_duration_in_sec = 0.25
+    overlap_ratio = 0.75
+
+    wanted_sampling_rate_Hz = 2000
+    downsampling_ratio = round(SampleMotorDataLimerick.sensor_sr_Hz /
+                               wanted_sampling_rate_Hz)
+
+    # ds_ratio = 10,  sr: 20K / 10 = 2000, 0.25 sec window, fft input will have: 500 samples,
+    # fftout's first 256 samples will be used
+    # cnn input will have 2556 samples
+
+    accel_in_second_dim = True
+
+    random_or_speed_split = True
+
+    return samplemotordatalimerick_get_datasets(data, load_train, load_test,
+                                                downsampling_ratio=downsampling_ratio,
+                                                signal_duration_in_sec=signal_duration_in_sec,
+                                                overlap_ratio=overlap_ratio,
+                                                eval_mode=eval_mode,
+                                                label_as_signal=label_as_signal,
+                                                random_or_speed_split=random_or_speed_split,
+                                                accel_in_second_dim=accel_in_second_dim)
+
+
+def samplemotordatalimerick_get_datasets_for_eval_with_signal(data,
+                                                              load_train=True,
+                                                              load_test=True):
+    """"
+    Returns Sample Motor Data Limerick Dataset For Evaluation Mode
+    Label is signal
+    """
+
+    eval_mode = True   # Test set includes validation normals
```

Comment:
comment would better be: testset now includes anormal samples as well as validation normals